modules/claims_api/app/controllers/claims_api/v1/forms/power_of_attorney_controller.rb

```diff
@@ -87,6 +87,8 @@ def status
         #
         # @return [JSON] Last POA change request through Claims API
         def active
+          validate_user_is_accredited! if header_request?
```

Comment:
If I understand the logic better now, it doesn't seem like there needs to be this same check in v0 because a veteran  checking their own active POA doesn't need this same validation, right?

Reply:
Well, v0 doesn't really have any knowledge of a particular user and there is explicit trust in the system that has production access. So this check would only be possible to perform on v1.